app/models/state_file_az_intake.rb

```diff
@@ -103,13 +104,15 @@ class StateFileAzIntake < StateFileBaseIntake
   enum eligibility_married_filing_separately: { unfilled: 0, yes: 1, no: 2 }, _prefix: :eligibility_married_filing_separately
   enum eligibility_529_for_non_qual_expense: { unfilled: 0, yes: 1, no: 2 }, _prefix: :eligibility_529_for_non_qual_expense
   enum made_az321_contributions: { unfilled: 0, yes: 1, no: 2 }, _prefix: :made_az321_contributions
+  enum made_az322_contributions: { unfilled: 0, yes: 1, no: 2 }, _prefix: :made_az322_contributions
   enum eligibility_lived_in_state: { unfilled: 0, yes: 1, no: 2 }, _prefix: :eligibility_lived_in_state
   enum eligibility_out_of_state_income: { unfilled: 0, yes: 1, no: 2 }, _prefix: :eligibility_out_of_state_income
 
   validates :made_az321_contributions, inclusion: { in: ["yes", "no"]}, on: :az321_form_create
+  validates :made_az322_contributions, inclusion: { in: ["yes", "no"]}, on: :az322_form_create
   validates :az321_contributions, length: { maximum: 10 }
+  validates :az322_contributions, length: { maximum: 10 }, on: :az322_form_create
```

Comment:
This is non-blocking and maybe trivial, but do we need to restrict this validation to this context?

Reply:
possibly not! i kept it there because it was already context-restricted (i just renamed the context). i guess the worry with not restricting it would be that someone would hit this validation error in a context where it can't be displayed and see a 500 or something like that. but those contexts don't exist as far as i know so happy to remove it